dizoo/procgen/coinrun/envs/coinrun_env.py

```diff
@@ -1,31 +1,33 @@
 from typing import Any, List, Union, Optional
```

Comment:
can you merge `coinrun`, `maze`, `bigfish` into a unified `ProcgenEnv` with different env_id args